jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -546,8 +548,14 @@ public ColumnSortList reloadColumnSortList() {
                         .findFirst();
                 if (optional.isPresent()) {
                     Column col = optional.get().getColumn();
-                    col.setDataStoreName(gridSortedColumnPreference.getDataStoreName());
-                    columnSortList.push(new ColumnSortList.ColumnSortInfo(col, gridSortedColumnPreference.isAscending()));
+                    String dataStoreName = gridSortedColumnPreference.getDataStoreName();
+                    col.setDataStoreName(dataStoreName);
```

Comment:
@bxf12315 in that case should we avoid adding to columnSortList if the column is already there?

Reply:
@cristianonicolai I think yes, but I can't find exactly reason.

Reply:
@bxf12315 I think you can just check the name of the columns in the columnSortList. If this new column is already there, just skip adding again.

Reply:
@cristianonicolai I try to skip it, code like this:
if(!columnSortList.get(0).getColumn().getDataStoreName().equals(col.getDataStoreName()) || !columnSortList.get(0).isAscending() == gridSortedColumnPreference.isAscending()){
                        columnSortList.push(new ColumnSortList.ColumnSortInfo(col, gridSortedColumnPreference.isAscending()));
                    }
but there also throw Error: IndexOutOfBoundsException: Row index: 0, Row size: 6.
I also try to clear ColumnSortedList before add "Createon", it is also throw same exception.